python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-adaptive-virtual-topology.j2

```diff
@@ -53,6 +53,28 @@ router adaptive-virtual-topology
 {%         if profile.load_balance_policy is arista.avd.defined %}
       path-selection load-balance {{ profile.load_balance_policy }}
 {%         endif %}
+{%         if profile.metric_order is arista.avd.defined %}
+      metric order {{ profile.metric_order }}
+{%         endif %}
+{%         if profile.outlier_elimination.enabled is arista.avd.defined(true) %}
+      path-selection outlier elimination
+{%         elif profile.outlier_elimination.enabled is arista.avd.defined(false) %}
+      path-selection outlier elimination disabled
+{%         endif %}
```

Comment:
it shows only disabled command in the running-config

```
wan1-dc1(config-profile-scavenger)#path-selection outlier elimination disabled 
wan1-dc1(config-profile-scavenger)#show active 
router adaptive-virtual-topology
   profile scavenger
      path-selection outlier elimination disabled
wan1-dc1(config-profile-scavenger)#path-selection outlier elimination 
wan1-dc1(config-profile-scavenger)#show active 
router adaptive-virtual-topology
   profile scavenger
wan1-dc1(config-profile-scavenger)#ex
```

Reply:
updated the key name as disabled and rendering only disabled command